media-nep/src/test/java/nl/vpro/nep/service/impl/NEPItemizeServiceImplITest.java

```diff
@@ -63,11 +63,20 @@ public void itemize() throws IOException, NEPException {
     @Test
     @Order(10)
     @Tag("dvr")
-    public void itemizeDvr() throws NEPException {
+    public void itemizeDvr() throws NEPException, InterruptedException {
         Instant start = Instant.now();
         NEPItemizeServiceImpl itemizer = new NEPItemizeServiceImpl(NEPTest.PROPERTIES);
         response = itemizer.itemizeLive("npo-1dvr", Instant.now().minusSeconds(300), Instant.now().minusSeconds(60), null);
         log.info("response: {} {}", response, start);
+        while(true) {
+            ItemizerStatusResponse jobs = itemizer.getItemizerJobStatus(response.getId());
+            Thread.sleep(1000);
```

Comment:
Moet de sleep niet naar onderen?

Reply:
Nee, het zit na de eerste call. Het is onzin om eerst te wachten lijkt me.

Reply:
je hebt gelijk